### ISSUE TITLE
Dosbox-x always opens in window mode when launched from a batch file

### ISSUE THREAD

**Issue by Focus78** [joncampbell123/dosbox-x#2810]
A reply would be nice?

**Comment by joncampbell123** [comment]
This is Windows, correct?

DOSBox-X.exe is a graphical application, so Windows will always launch it as it's own window and detach it from the command line. If it were compiled as a command line application, it would work, but then running from the desktop would also bring up a console window.

**Comment by Focus78** [comment]
What?

Can you just fix this so that "fullscreen = true" in the config file actually functions correctly?

Of fix the SDL2 soundfont problem since this behaviour does not take place in the build...

Either would solve the problem.

**Comment by maron2000** [comment]
Hi, I started dosbox-x from a simple batch file `test.bat` and it started in full screen mode.
Are there any difference in setting or procedure with yours?
I tested on Windows SDL1, and SDL2 both 32-bit. 
(I believe you prefer 32-bit version, but would you specify which OS and version you tried if it isn't the case)

test.bat
```
dosbox-x
```

conf file
```
fullscreen        = true      #changed from default false to true
fulldouble        = true      #changed from default false to true (may not affect the result though)
fullresolution    = desktop
```

**Comment by Focus78** [comment]
joncampbell123 already stated that he was able to replicate the problem but has since done nothing about it.

I'm not going to repeat the same process all over again. You can ask him for details or see the other report if you have an unlimited amount of time on your hands to test what's already been verified.

**Comment by maron2000** [comment]
Oh, this was a duplicate of https://github.com/joncampbell123/dosbox-x/issues/2643?
I read it and understood that this is a compatibility issue of Windows SDL1 version and GlovePIE.
Please correct me if anything is wrong.

**Comment by maron2000** [comment]
I managed to install GlovePIE 0.45 and confirmed the phenomenon.
It seems that if you wait to GlovePIE to completely launch before starting DOSBox-X, it works.
(So maybe not only GlovePIE but launching other software as well)

The example below waits for 2 seconds, you are free to tune it.  
It worked for me, so it's worth trying.  Hope it helps.

```
start piefree -test.pie
timeout /t 2
start dosbox-x -set fullscreen=true
```

**Comment by Focus78** [comment]
@maron2000

I can confirm that "timeout /t X" added to the batch file produces the expected behavior. Now why it's necessary or why joncampbell123 brushed it off when it ended up being such a simple workaround is another story...

Thank you.
